clubs/serializers.py

```diff
@@ -170,23 +178,50 @@ def get_image_url(self, obj):
         else:
             return self.context['request'].build_absolute_uri(obj.image.url)
 
+    def get_parent_orgs(self, obj):
+        return []
+        # return [c.slug for c in obj.parent_orgs.all()]
+
     def create(self, validated_data):
         """
         Manual create method because DRF does not support writable nested fields by default.
         """
         # assign tags to new club
-        has_tags = 'tags' in validated_data
+        tags = None
+        parent_orgs = None
+        badges = None
 
-        if has_tags:
+        if 'tags' in validated_data:
             tags = []
             for tag in validated_data.pop('tags'):
                 tags.append(Tag.objects.get(**tag))
 
+        # Don't let clubs add parent orgs for now; they will be added through the admin
+        # interface.
+
+        # if 'parent_orgs' in validated_data:
+        #     parent_orgs = []
+        #     for parent_org in validated_data.pop('parent_orgs'):
+        #         parent_orgs.append(Club.objects.get(**parent_org))
+
+        if 'badges' in validated_data:
+            badges = []
+            for badge in validated_data.pop('badges'):
+                badges.append(Badge.objects.get(**badge))
+
+        # Note: it's important that all nested fields are POPPED from the validated data at this point. Otherwise,
+        # DRF will throw an error calling create on the superclass.
         obj = super().create(validated_data)
 
-        if has_tags:
+        if tags is not None:
             obj.tags.set(tags)
 
+        if parent_orgs is not None:
```

Comment:
`parent_orgs` will always be None with the commented out code. Should probably just delete this block depending on the response to my above comment.